serverless/serverless
# Services

## What did you implement:

Integration with the Serverless Platform to publish service information

## How can we verify it:

Run `serverless login` and then deploy any service. You should see a log message that you service is available at the Platform.



## Todos:

- [x] Write documentation (no since we don't we want to try it only with interested users for now)

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
